src/server/rdb_save.cc

```diff
@@ -688,6 +689,20 @@ error_code RdbSerializer::SendFullSyncCut(io::Sink* s) {
   return FlushToSink(s);
 }
 
+error_code RdbSerializer::WriteJournalEntries(absl::Span<const journal::Entry> entries) {
+  // TODO: a base::IoBuf shold be really a sink...
+  io::StringSink ss{};
+  JournalWriter writer{&ss};
+  for (const auto& entry : entries) {
+    RETURN_ON_ERR(writer.Write(entry));
```

Comment:
We should maybe pass the current db index here to eliminate the first select entry